concepts/observability-strategy/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,89 @@
+With a stripped-down image version basic tests could be performed to assure that standard scenarios are working and also a security triaging is realistic. However, there will be still not the full guarantee of that an update of your collector instance will be working reliably.
+Providing the base image only brings no value as you rarely can use it without any additional processors. So users would need to opt-out always (which is possible without problems) but again the value of the whole feature is under question.
+
+### The power of auto-instrumentation
+
+Auto-instrumentation sounds like a great idea at a first view. It will provide so much insights of your application without any investment. However, that approach has a high chance of also breaking your application. Especially for nodeJS, the auto feature will just add a node package to an application which can cause version conflicts and block startups.
+Instead, users should add instrumentation (even automatic ones) before the app gets deployed, so that this happens in a controlled and tested way. Again, think of updates to the auto-instrumentation library. If that happens at runtimes, suddenly an application might not startup anymore.
+
+### Injection of settings
+
+The injection of the typical and recommended setting for a workload instrumented by the OTEL SDK is very useful and lowers the burden for managing an application. Insteaf of injecting a hardcoded URL which might change over time, the operator will take care by annotation to inject the proper URL.
+
+## Conclusion
+
+The proposed solution has many advantages. The standard way of auto-injection will be working instantly without any kyma-specific invention. The kyma specific operator can be simplified and users can run more easily custom collector instances.
+On the negative side, the feature will introduce mainly a technology, not a feature. It will promise to take care of the full lifecycle of custom collectors but cannot deliver that promise. Updates of custom instances cannot be guaranteed to be reliable. Also, the included auto-instrumentation feature can break your application on updates of the instrumentation logic.
+
+As the upgrade topic is too serious, the proposed solution cannot be followed in that way. Instead, a compromise seems as a better solution by adopting the very good concept of env variable injection:
+1. Assure at anytime that users can install an own OTEL operator at anytime to manage custom collectors more easy, and also to have instrumentation if desired. The benefit of having the operator and collector update automatically is not provided with that.
```

Comment:
The numbered list implies that the order of the options matters. Is that true?
If they're alternatives to each other, I would suggest a bullet list.


```suggestion
1. Assure at any time that users can install an own OTEL operator at any time to manage custom collectors more easily, and also to have instrumentation if desired. The benefit of having the operator and collector update automatically is not provided with that.
```